mergify_engine/duplicate_pull.py

```diff
@@ -29,6 +29,29 @@
     ["reference already exists", "You may want to first integrate the remote changes"]
 )
 
+MERGIFY_PULL_REQUEST_DOC = """
+---
+
+<details>
+<summary>Mergify commands and options</summary>
+<br />
+More conditions and actions can be found in the [documention](https://doc.mergify.io/).
+<br />
+<br />
+You can also trigger Mergify actions by commenting on pull requests:
+
+- `@Mergifyio rebase` will rebase this PR
+- `@Mergifyio backports <destination>` will backport this PR on `<destination>` branch
+
+Additionally, on Mergify [dashboard](https://dashboard.mergify.io/) you can:
+
+- look at your merge queues
+- generate the `.mergify.yml` with the simulator.
```

Comment:
maybe use `configuration file` rather than `.mergify.yml` since it can also be named `.mergify/config.yml`?